### DISCUSSION TITLE
Save issues

### DISCUSSION THREAD

**Discussion by itaysk** [community/community#58161] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I get a notification I can "save" it which will appear under the "saved" category. I'd like to save things outside the notifications flow. For example, I may have "unsaved" something accidentally and want to readd it. Or I want to save something I wasn't notified about.
I expect to be able to go to a issue/pr/discussion and have an option to "save" it directly.

**Comment by Gustavonn07** [comment]
As far as i know the only way to save things on github would be by:

Stars: GitHub allows you to star repositories. This action is similar to "favoriting" a repository and allows you to easily access them later. Stars are located at the top right of a repository.

Bookmarks: In addition to stars, you can also use the browser's bookmarking feature to save important GitHub pages. Just click your browser's bookmark button when visiting the page you want to save.

Reading List: If you're using GitHub's projects feature, you can create a column called "Reading List" in your project and add links to important pages or discussions you want to follow. This allows you to easily organize and access relevant content.

But if you want to save notifications I don't think github has something specific for what you're talking about, however the notifications are sent to your saved email, so maybe you can save there.

Hope it helped!

**Reply by itaysk** [reply]
> if you want to save notifications I don't think github has

GitHub does have this already. What I'm asking for is to be able to save things that didn't arrive as notification

**Comment by ia** [comment]
Hello. I do support the request for this feature. In fact, lacking of it is getting pretty annoying. It's so super convenient to save a particular issue or discussion with very interesting conversation, but not when you see it outside of _Notifications_ interface. It just doesn't make much of a sense. Please, add `Save` button (with the same style & art which you have in _Notifications_) inside _Issue_ and _Discussion_ interface. :pray:
